js/gui.js

```diff
@@ -286,9 +286,10 @@ GUI_control.prototype.content_ready = function (callback) {
         log(this); // ...print out "this", which now refers to each ".cf_tip" DOM element
     });
 
+    //Note: using i18n_title, so no need for convoluted content: $(this).children('.cf_tooltiptext')
     $('.cf_tip').each(function() {
         $(this).jBox('Tooltip', {
-            content: $(this).children('.cf_tooltiptext'),
+           // content: $(this).children('.cf_tooltiptext'),
```

Comment:
delete unused code rather than commenting it out.  We can use git history if we ever want it back.